src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -161,101 +158,95 @@ public void onDye(PlayerInteractEntityEvent e) {
     }
 
     @EventHandler
-    public void onMiningDrillUse(PlayerInteractEvent e) {
-        final Block block = e.getClickedBlock();
+    public void onDrillUse(PlayerInteractEvent event) {
+        final Block block = event.getClickedBlock();
         if (block == null) {
             return;
         }
 
+
         final Material blockType = block.getType();
         final Location blockLocation = block.getLocation();
+        final ItemStack hand = event.getItem();
+
+        if (!SlimefunTag.STONE_VARIANTS.isTagged(blockType)) {
```

Comment:
STONE_VARIANTS do not inclide any of the newer stone-style blocks which, likely shouldn't be tackled here, but you may wanna PR to make this tool better? Thats just IMO though, I guess.

Reply:
mmmh so pr to improve STONE_VARIANTS or add deepslate variants?

Reply:
I mean, improving STONE_VARIANTS would benefit more, imo :)

Reply:
will do